test/cassava/client_test.rb

```diff
@@ -161,6 +161,25 @@ def string_keys(hash)
       end
     end
 
+    context 'select_ttl' do
```

Comment:
Would be good to see one more test with two where_arguments (one integer and one string) just to fully exercise that `select_ttl_statement` method ...